pkgs/build-support/cc-wrapper/ld-wrapper.sh

```diff
@@ -145,6 +145,10 @@ if [ "$NIX_@infixSalt@_DONT_SET_RPATH" != 1 ]; then
             fi
         done
     done
+
+    if [ -n "${NIX_COREFOUNDATION_RPATH:-}" ]; then
```

Comment:
Oh, I missed this before, but this should have probably done the `@infixSalt@` thing, to support darwin->darwin cross compilation where the path may not be the same.

Reply:
I wrote this before the salts where added. Not sure if it really matters since it will only point to `/System/Library/Frameworks`, similar to the dyld path with `LD_DYLD_PATH`.

Reply:
Oh, my bad @LnL7. I thought this was from the PR you just merged in the last 24 hours, and never double checked!